spec/models/users_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  describe 'validations' do
+    it { should validate_presence_of(:name) }
+    it { should validate_presence_of(:email) }
+    it { should validate_length_of(:name).is_at_most(20) }
+  end
+
+  describe 'association' do
+    it { should have_many(:posts) }
+    it { should have_many(:comments) }
+    it { should have_many(:likes) }
+    it { should have_many(:invitations) }
```

Comment:
- [ ] There are some association and instance methods in user model that are not tested yet -- Let's add them

Also if you have decided to move the `timeline_posts` method in user model don't forget to add its tests.